# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/DocumentL10n.cpp
# Commit: 626b40e3aad1
# Full Hash: 626b40e3aad147d59a439b0bd1d5dbd328ad6b1a
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-02 21:43:56
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 1cbab987bb7d -r 626b40e3aad1 dom/l10n/DocumentL10n.cpp
--- a/dom/l10n/DocumentL10n.cpp	Mon Aug 02 09:54:10 2021 +0000
+++ b/dom/l10n/DocumentL10n.cpp	Mon Aug 02 09:54:10 2021 +0000
@@ -48,7 +48,7 @@
 }
 
 DocumentL10n::DocumentL10n(Document* aDocument, bool aSync)
-    : DOMLocalization(aDocument->GetScopeObject(), aSync, {}),
+    : DOMLocalization(aDocument->GetScopeObject(), aSync),
       mDocument(aDocument),
       mState(DocumentL10nState::Constructed) {
   mContentSink = do_QueryInterface(aDocument->GetCurrentContentSink());